### PR TITLE
Added contribution.md file

### PR DESCRIPTION
**Pull request by VyjayantiN**
## Summary
Added a Contribution.md file to provide guidelines for contributing to the project. This file will help new contributors understand the process and requirements for making contributions. No dependencies are required for this change.

## Fixes #
Fixed the issue #341 

## Type of change
Please select the relevant option.
- [ ] Bug fix
- [ ] New feature
- [ ] Breaking change
- [x] Documentation update
- [ ] Other (please describe)

## Checklist
Please tick the relevant options.
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by VyjayantiN** [comment]
@AbhiPatel10 @Neilblaze Please check this out.

File: contribution.md
Comment (Neilblaze) [inline contribution.md:1]: cc: @kartikayasija mind taking a look at this?

**Comment by kartikayasija** [comment]
@VyjayantiN  Fix this issue in the Footer , 
image
image

**Comment by VyjayantiN** [comment]
@kartikayasija I didn't make any improper changes to the footer. Are you asking me to look into them as a mentee or did I make any mistake?  However, I would love to change that.

**Comment by VyjayantiN** [comment]
@Neilblaze @kartikayasija Please check this out!!

**Comment by kartikayasija** [comment]
Just remove list-style: circle, then I'll merge.
image

**Comment by VyjayantiN** [comment]
@kartikayasija please check out the merged pull request #355  regarding the footer. I changed it long back and you already merged it @kartikayasija .
This is pr #345 regarding the contribution.md
Please check out both the PRs to avoid confusions!!

**Comment by Neilblaze** [comment]
@VyjayantiN It would be advisable to include the specific `#ID` linking to the corresponding issue for every PR you submit.

**Comment by kartikayasija** [comment]
> @kartikayasija please check out the merged pull request #355 regarding the footer. I changed it long back and you already merged it @kartikayasija . This is pr #345 regarding the contribution.md Please check out both the PRs to avoid confusions!!

Yes, the footer issue was resolved previously, but you have again commited that line in this PR, so when I will merge it, the issue will occur again. Just remove that one line instead of writing replies.

**Comment by VyjayantiN** [comment]
@kartikayasija which one line? I didn't understand

**Comment by Neilblaze** [comment]
@VyjayantiN 👇🏼  https://github.com/AbhiPatel10/AISKCON-CONSTRUCTION/blob/b85e1a656d73b18c0ef21dd4f609afd1eae915b1/aboutUs/style.css#L18

**Comment by VyjayantiN** [comment]
@Neilblaze Okay!!

**Comment by VyjayantiN** [comment]
I resolved that but it is showing that there are conflicts to merge. I am not understanding why it happened.

**Review by Neilblaze — APPROVED** [review]
SGTM! 👍🏼

**Comment by VyjayantiN** [comment]
Thank you @Neilblaze 🙏🙏🙏

**Comment by kartikayasija** [comment]
Resolve conflicts.

**Pull request closed without merging**